### PR TITLE
smallbank with flatbuffers

### PR DESCRIPTION
**Pull request by olgavrou**

### CODE REVIEW

File: samples/apps/smallbank/app/accounts.fbs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (ashamis) [inline samples/apps/smallbank/app/accounts.fbs:1]: Can all the fbs tx types files be combined?

File: samples/apps/smallbank/app/flatbuffer_wrapper.h [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+public:
+  std::unique_ptr<flatbuffers::DetachedBuffer> get_detached_buffer()
+  {
+    return std::make_unique<flatbuffers::DetachedBuffer>(builder.Release());
+  }
+
+  CBuffer get_buffer()
+  {
+    return {builder.GetBufferPointer(), builder.GetSize()};
```
Comment (ashamis) [inline samples/apps/smallbank/app/flatbuffer_wrapper.h:25]: Do we need to create a new vector here? can pointers and size be used?

Comment (olgavrou) [reply samples/apps/smallbank/app/flatbuffer_wrapper.h:25]: It's not a vector, it is a CBuffer, i.e. pointers and vectors _are_ used

File: samples/apps/smallbank/app/accounts.fbs (hunk 1 above)
Comment (olgavrou) [reply samples/apps/smallbank/app/accounts.fbs:1]: Yes I believe they can be combined, it is a matter of how we want to create our flatbuffer files per app. The combination would result in one file with all the potential fields, only some being populated per RPC, and one serializer/deserializer. I think it is a bit cleaner to have them separate, but I don't have a very strong opinion :)

**Comment by ashamis** [comment]
Maybe what I am saying is wrong but the speedup in the sign bft small bank test is concerning. Since the sig test now has the same throughput as the non signing variant, is signing broken?

**Comment by olgavrou** [comment]
@Alex signing was broken but it looks like it is fixed now, no?

**Pull request closed without merging**